yt_dlp/extractor/franceculture.py

```diff
@@ -44,18 +54,42 @@ def _real_extract(self, url):
             ''',
             webpage, 'video data'))
 
-        video_url = video_data.get('data-url') or video_data['data-asset-source']
-        title = video_data.get('data-asset-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
-
+        title = video_data.get('data-emission-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
         description = self._html_search_regex(
             r'(?s)<div[^>]+class="intro"[^>]*>.*?<h2>(.+?)</h2>',
             webpage, 'description', default=None)
+
+        # page has playlist
+        if (re.search(r'<div[^>]+class="[^"]*?podcast-list[^"?]*?"[^>]*>', webpage) is not None):
+            playlist_data = self._search_regex(
+                r'''(?sx)
+                    <div[^>]+class="[^"]*?podcast-list[^"?]*?"[^>]*>
+                    (.*?)
+                    <div[^>]+class="[^"]*?see-more-anchor[^"]*?">
+                ''',
+                webpage, 'playlist data')
+
+            entries = [
+                self.url_result(video_url, None, video_id, video_title)
+                for video_url, video_id, video_title in re.findall(
+                    r'''data-url="([^"]+)"[^>]*data-diffusion-path="([^"]+)"[^>]*data-diffusion-title="([^"]+)"''',
```

Comment:
U can use single quotes here instead of `'''`

Reply:
Done :)